### DISCUSSION TITLE
I have been rejected for my location

### DISCUSSION THREAD

**Discussion by Aftondil** [community/community#148048] [GitHub Education]
### Hi everyone,

i have a problem to get verified, i study at university of Wolverhampton tashkent campus (name: IDU), there is a question why are you located so far from campus, I shouldnt be at UK to get verified right?

**Comment by davevad93** [comment]
Hi @Aftondil ,

If you are located near your campus and are unable to apply, it's possible that GitHub has incorrect information about your school. Try contacting support by opening a ticket [here](https://support.github.com/contact/education), select "`My selected school might have incorrect or incomplete information e.g. domains or campus location`" and explain your issue.
